pkgs/development/tools/wasm-pack/default.nix

```diff
@@ -2,32 +2,31 @@
 , fetchFromGitHub
 , rustPlatform
 , pkgconfig
-, openssl
+, libressl
 , curl
 , Security
 }:
 
 rustPlatform.buildRustPackage rec {
   pname = "wasm-pack";
-  version = "0.8.1";
+  version = "0.9.1";
 
   src = fetchFromGitHub {
     owner = "rustwasm";
     repo = "wasm-pack";
     rev = "v${version}";
-    sha256 = "1z66m16n4r16zqmnv84a5jndr5x6mdqdq4b1wq929sablwqd2rl4";
+    sha256 = "1rqyfg6ajxxyfx87ar25nf5ck9hd0p12qgv98dicniqag8l4rvsr";
   };
 
-  cargoSha256 = "0hp68w5mvk725gzbmlgl8j6wa1dv2fydil7jvq0f09mzxxaqrwcs";
+  cargoSha256 = "095gk6lcck5864wjhrkhgnkxn9pzcg82xk5p94br7lmf15y9gc7j";
 
   nativeBuildInputs = [ pkgconfig ];
 
-  buildInputs = [ openssl ]
+  buildInputs = [ libressl ]
```

Comment:
could you add a comment here with the explanation, its not common at all that we choose a different ssl library.

Reply:
An oversight on my part. Added explanation in 40297b79c6daa6ed194fe783972d29e66253c485

Thanks for the review!